src/pkg/packager/deploy.go

```diff
@@ -109,8 +118,28 @@ func Deploy(ctx context.Context, pkgLayout *layout.PackageLayout, opts DeployOpt
 		return DeployResult{}, err
 	}
 
+	if len(opts.Values) > 0 && !feature.IsEnabled(feature.Values) {
+		return DeployResult{}, fmt.Errorf("package-level values passed in but \"%s\" feature is not enabled."+
+			" Run again with --features=\"%s=true\"", feature.Values, feature.Values)
+	}
+
+	// Read the default package values off of the pkgLayout.Pkg.Values.Files.
+	// Resolve values file paths relative to the package directory
+	valueFilePaths := make([]string, len(pkgLayout.Pkg.Values.Files))
+	for i, vf := range pkgLayout.Pkg.Values.Files {
+		valueFilePaths[i] = filepath.Join(pkgLayout.DirPath(), vf)
+	}
+	vals, err := value.ParseFiles(ctx, valueFilePaths, value.ParseFilesOptions{})
+	if err != nil {
+		return DeployResult{}, err
+	}
+	// Package defaults are overridden by deploy values.
+	vals.DeepMerge(opts.Values)
+	logger.From(ctx).Debug("package values", "values", vals)
```

Comment:
nit: we have `l` for use:

```suggestion
	l.Debug("package values", "values", vals)
```

Reply:
Revised each of these in latest